frontend/src/views/main/project/ProjectsHome.vue

```diff
@@ -47,7 +47,7 @@
         </v-row>
       </v-card>
       <v-hover v-slot="{ hover }" v-for="p in projects" :key="p.id">
-        <v-card outlined tile class="grey lighten-5 project" :class="[{ 'info': hover }]" :to="{ name: defaultRoute, params: { id: p.id } }" v-show="creatorFilter === 0 || creatorFilter === 1 && p.owner.id === userProfile.id || creatorFilter === 2 && p.owner.id !== userProfile.id">
+        <v-card outlined tile class="grey lighten-5 project" :class="[{ 'info': hover }]" :to="{ name: defaultRoute, params: { id: p.id } }" v-show="creatorFilter === 0 || creatorFilter === 1 && p.owner.id === userProfile.id || creatorFilter === 2 && p.owner.id !== userProfile.id" @click="() => { updateCurrentProject(p.id) }">
```

Comment:
I think that we shouldn't mix up click and to but add a router push in the updateCurrentProject method to make it easy to maintain